### PR TITLE
Davidlee0602/commentrank api

### PR DESCRIPTION
**Pull request by davidlee0602**
API for comments and ratings submitted by user.

### CODE REVIEW

**Review by dlimx — DISMISSED** [review]
LGTM

File: client/components/RateModal.jsx [hunk 1]
```diff
@@ -33,29 +37,62 @@ export const RateModal = ({ setRate }) => {
...
+            <Grid container justify="flex-end">
+              <Button
+                variant="contained"
+                color="primary"
+                onClick={() => {
+                  handleClose();
+                  if (rateVal > 0 || commentVal.length > 0) {
+                    api
+                      .post('/quizzes/rating', {
+                        User: currentUser.uid,
```
Comment (dlimx) [inline client/components/RateModal.jsx]: Wait sorry, let's use the User from the API's middleware rather than passing it in

Comment (davidlee0602) [reply client/components/RateModal.jsx]: okay! fixed! Utilizing auth middleware passed as a param into the rating route

**Review by ddvega — APPROVED** [review]
Looks great. Nice job!

**Pull request merged**